controllers/cartController.js

```diff
@@ -0,0 +1,63 @@
+const cartService = require('../services/cartService');
+const { catchError } = require('../middlewares/error');
+
+const createCartItem = catchError(async (req, res) => {
+  try {
+    const { productId, quantity } = req.body;
+
+    const userId = req.user.id;
+
+    const cartItem = await cartService.createCartItem({
+      userId,
+      productId,
+      quantity,
+    });
+    res.status(201).json({ message: cartItem });
+  } catch (err) {
+    res.status(400).json({ message: err.message });
+  }
+});
+
+const getCartList = async (req, res) => {
+  try {
+    const userId = req.user.id;
+    const cartList = await cartService.getCartList(userId);
+    res.status(201).json({ cartList });
+  } catch (err) {
+    res.status(400).json({ message: err.message });
+  }
+};
+
+const updateCart = catchError(async (req, res) => {
+  try{
+  const userId = req.user.id;
+
+  const { productId, quantity } = req.body;
+
+  const update = await cartService.updatedCart({ userId, productId, quantity });
+
+  res.status(201).json({ message: update });
+  } catch (err) {
+    res.status(400).json({ message: err.message });
+  }
+});
+
+const deleteCart = catchError(async (req, res) => {
+  try {
+    const userId = req.user.id;
+    const { productId } = req.params;
+
+    const deleteCart = await cartService.deleteCart({ userId, productId });
+
+    res.status(201).json({ message: deleteCart });
```

Comment:
```suggestion
    res.status(204).send();
```
위에서 언급드린 코멘트와 마찬가지로 삭제(DELETE) 메소드와 더 적합한 status code가 있습니다. 이를 활용하여 주시고 204는 클라이언트에서 바디를 받을 수 없기 때문에 따로 메세지를 담지 않고 위처럼 .send() 메소드를 통해 response를 보내주기만 해도 됩니다.